python/GafferOSLUI/OSLShaderUI.py

```diff
@@ -57,6 +57,11 @@ def __nodeDescription( node ) :
 
 def __nodeIcon ( node ) :
 
+	if node["name"].getValue().startswith( "as_" ) :
+		# Appleseed advertises icons that it doesn't seem
```

Comment:
Yes. Thanks for the fix. The icons are currently in appleseed.maya repo. We have to move them to the main repo.